packages/core/src/index.ts

```diff
@@ -1,24 +1,25 @@
 //
 // Copyright © 2020, 2021 Anticrm Platform Contributors.
-// 
+//
 // Licensed under the Eclipse Public License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License. You may
 // obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
-// 
+//
 // Unless required by applicable law or agreed to in writing, software
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// 
+//
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
 
 export * from './classes'
 export * from './tx'
 export * from './storage'
+export * from './processor_doc'
```

Comment:
Please can we move `Processor` to `tx.ts` or remove underscore (all our `ts` files are without underscores) and I also do not think that `_doc` suffix makes things better to understand

Reply:
This is caused by next PR for vdocs, since it will be one more processor. Will move to tx.

Reply:
Moved to tx. Thanks.